graphql/resolvers.go

```diff
@@ -1569,15 +1569,24 @@ func (r *queryResolver) PatchTasks(ctx context.Context, patchID string, sorts []
 			taskSorts = append(taskSorts, task.TasksSortOrder{Key: key, Order: order})
 		}
 	}
+
+	v, _ := r.sc.FindVersionById(patchID)
+	includeInactiveTasks := false
+	// Query inactive tasks for inactive versions
```

Comment:
We want to query inactive tasks for inactive mainline commit versions so a user can schedule them from the UI. Since the tasks for mainline commit versions have never ran, we need to adjust when [this line](https://github.com/evergreen-ci/evergreen/pull/5136/files#diff-4b29d204064065e99ec7762e466f51f0ac8461fcc5f8fc2eeda53d87a5044060R3139) is ran so it's only executed when `patchID` represents an inactive mainline commit version. I think this will not break any changes related to filtering out generated tasks because an inactive mainline commit version has not yet ran and won't have any generated tasks.